Sources/ArgumentParser/Parsable Types/CommandConfiguration.swift

```diff
@@ -61,6 +68,9 @@ public struct CommandConfiguration {
   ///   - commandName: The name of the command to use on the command line. If
   ///     `commandName` is `nil`, the command name is derived by converting
   ///     the name of the command type to hyphen-separated lowercase words.
```

Comment:
For the current behavior, I think we need to amend this to say that when this parameter is `nil` the command name is derived by converting the type name or using the executable.

Reply:
Yep, I agree. I think this would work:
```suggestion
  ///     the name of the command type to hyphen-separated lowercase words
  ///     or the executable name if `shouldUseExecutableName` is `true`.
```